deployments/optimism/usdc/configuration.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85",
+  "baseTokenPriceFeed": "0x16a9FA2FDa030272Ce99B29CF780dFA30361E0f3",
+  "pauseGuardian": "0x3fFd6c073a4ba24a113B18C8F373569640916A45",
+  "borrowMin": "1e5",
+  "storeFrontPriceFactor": 0.6,
+  "targetReserves": "20000000e6",
+  "rates": {
+    "supplyKink": 0.9,
+    "supplySlopeLow": 0.059,
+    "supplySlopeHigh": 2.9,
+    "supplyBase": 0,
+    "borrowKink": 0.9,
+    "borrowSlopeLow": 0.061,
+    "borrowSlopeHigh": 3.2,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0e15",
+    "baseBorrowSpeed": "0e15",
+    "baseMinForRewards": "1000e6"
+  },
+  "assets": {
+    "WETH": {
+      "address": "0x4200000000000000000000000000000000000006",
+      "priceFeed": "0x13e3Ee699D1909E989722E753853AE30b17e08c5",
+      "decimals": "18",
+      "borrowCF": 0.83,
+      "liquidateCF": 0.9,
+      "liquidationFactor": 0.95,
+      "supplyCap": "0e18"
+    },
+    "WBTC": {
+      "address": "0x68f180fcCe6836688e9084f035309E29Bf0A2095",
+      "priceFeed": "0xD702DD976Fb76Fffc2D3963D037dfDae5b04E593",
+      "decimals": "8",
+      "borrowCF": 0.8,
+      "liquidateCF": 0.85,
+      "liquidationFactor": 0.95,
+      "supplyCap": "0e8"
+    },
+    "OP": {
+      "address": "0x4200000000000000000000000000000000000042",
+      "priceFeed": "0x0D276FC14719f9292D5C1eA2198673d1f4269246",
+      "decimals": "18",
+      "borrowCF": 0.65,
+      "liquidateCF": 0.7,
+      "liquidationFactor": 0.8,
+      "supplyCap": "0e18"
+    },
+    "wstETH": {
+      "address": "0x1F32b1c2345538c0c6f582fCB022739c4A194Ebb",
+      "priceFeed": "0x698B585CbC4407e2D54aa898B2600B53C68958f7",
```

Comment:
This is the market-based `wstETH / USD` price feed. Lido's recent proposals to list wstETH seem to go with the exchange-rate-based `wstETH / stETH` * `ETH / USD` custom price feed. Let's maybe wait for them to finalize on that decision and make the same decision here to be consistent.

Reply:
Not clear. Could we not wait for Lido's decision?

Reply:
As you suggested, you could just exclude `wstETH` for now until that decision has been made.